packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -99,8 +108,15 @@ class CouldNotCreateFolderError(BaseCreateFolderError):
     msg_template = "Could not create folder='{folder}' and parent='{parent}'"
 
 
-class GroupIdDoesNotExistError(BaseCreateFolderError):
-    msg_template = "Provided group id '{gid}' does not exist "
+class GroupIDsDoNotExistError(BaseCreateFolderError):
+    msg_template = "Any of the folloing gids='{gids}' do not exist"
+
+
+class RootFolderRequiresAtLeastOnePrimaryGroupError(BaseCreateFolderError):
+    msg_template = (
+        "parent={parent} is None (please check) and gids={gids} did not contain a PRIMARY group. "
+        "Cannot create a folder isnide the 'root' directory."
+    )
 
```

Comment:
```suggestion
    msg_template = (
        "No parent={parent} defined and groupIDs={gids} did not contain a PRIMARY group. "
        "Cannot create a folder inside the 'root' directory."
    )

```
This error is also a bit weird. should you not separate them?